README.md

```diff
@@ -271,5 +301,36 @@ An implementation of the [middleware resolver for Unity](https://github.com/Shan
 Install-Package PipelineNet.Unity
 ```
 
+## Migrate from PipelineNet 0.10 to 0.20
+In PipelineNet 0.20, `Finally` overloads that use `Func` have been made obsolete.
```

Comment:
Most nuget packages use semantic versioning - meaning breaking changes are only allowed with major version bumps: `X._._`. Since this is pre-`1.0.0` I just took the next most significant number.

Let me know if you want this changed despite this though.